src/stories/Library/opening-hours/OpeningHours.tsx

```diff
@@ -0,0 +1,48 @@
+import { FC } from "react";
+import { ReactComponent as ArrowLeft } from "../../../public/icons/collection/ArrowLeft.svg";
+import { ReactComponent as ArrowRight } from "../../../public/icons/collection/ArrowRight.svg";
+import { GroupedOpeningHours } from "./OpeningHoursExampleData";
+import OpeningHoursWeekList from "./OpeningHoursWeekList";
+
+type OpeningHoursProps = {
+  title: string;
+  weekCurrentlyDisplayed: string;
+  groupedOpeningHours: GroupedOpeningHours;
+};
+
+const OpeningHours: FC<OpeningHoursProps> = ({
+  groupedOpeningHours,
+  weekCurrentlyDisplayed,
+}) => {
+  return (
+    <div className="opening-hours">
+      <div className="opening-hours__wrapper">
+        <div className="opening-hours__header">
+          <h2 className="opening-hours__heading">Åbningstider</h2>
+          <div className="opening-hours__navigation-controls">
+            <button
+              className="opening-hours__navigation-control"
+              aria-label="Previous week"
+              type="button"
+            >
+              <ArrowLeft />
+            </button>
+            <div className="opening-hours__week-display">
+              {weekCurrentlyDisplayed}
+            </div>
+            <button
+              className="opening-hours__navigation-control"
+              aria-label="Next week"
+              type="button"
+            >
+              <ArrowRight />
+            </button>
+          </div>
+        </div>
+        <OpeningHoursWeekList groupedOpeningHours={groupedOpeningHours} />
+      </div>
+    </div>
```

Comment:
Consider if we can drop the `div className="opening-hours__wrapper">` element.

We already have a `<div className="opening-hours">` element to wrap the entire component.

Reply:
I need the layout-container for opening_hours, but the contents need to be max-width:600px. within that. 

I wasn't sure whether to use a wrapper for all the content, or to add the max-width two places, in this case __header and __content. 

I have changed it to the second approach now, which effectively does the same, but without the wrapper. Not sure which i like best.